packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -21,6 +21,21 @@ export default class RatingBar extends PureComponent {
     maxScore: 5,
   }
 
+  get ratingLegend(){
+    const {
+      theme, 
+      score,
+      ratingValue
+    } = this.props
+
+    return (
+      <div className={theme.Legend}>
+        <div className={theme.RatingBar_RatingValue}> {ratingValue} </div>
+        <div className={theme.RatingBar_LegendIcon}></div>
+      </div>
```

Comment:
This assumes that the developer will want to display an icon. How about somthing like this, and let the developer pass in whatever they want?
```
<div className={theme.RatingBar_Prefix}>{this.props.prefixLabel}</div>
```
Then if I want an icon  I can just pass in whatever content I want.